EdgeOfEmpireBot/EdgeOfEmpireBot/Service/DataService.cs

```diff
@@ -80,6 +82,34 @@ public async Task WriteGameToFile(SteamGameDetails gameDetails)
             await File.WriteAllTextAsync(gameFilePath, JsonConvert.SerializeObject(games, Formatting.Indented));
         }
 
+        /// <summary>
+        /// Writes the phrase to the Remember.json file to recall it later
+        /// </summary>
+        public async Task AddRememberPhraseToFile(string command)
```

Comment:
There should probably be a check to ensure message length isn't too long. 
2000 characters is the limit. Keeping in mind we normally add some extra stuff to the message itself, be it formatting or HK47 so something less than 2000 characters, maybe 1900 to be safe?

Otherwise the remember files gonna have a bunch of potential text thats just gonna result in errors down the line. 

Might also be worth modifying for special characters, just in case, or to prevent our formatting breaking as well (I've done that to Joshs bot in the past)

Reply:
Yeah that sounds reasonable. I will keep this in mind when creating the router

Reply:
I've made #31 and #30  so we dont forget